src/NumericalAlgorithms/Interpolation/InterpolationTargetApparentHorizon.hpp

```diff
@@ -100,7 +100,6 @@ namespace Actions {
 ///
 /// Uses:
 /// - DataBox:
-///   - `::Tags::Domain<VolumeDim, Frame>`
```

Comment:
Doesn't the action still use ::Tags::Domain from the databox?  (The Domain gets into the DataBox via a different path than before, but one can still get it from the DataBox and use it).

Reply:
okay, I see, they are used indirectly via a function call...